Doc/faq/design.rst

```diff
@@ -279,14 +276,14 @@ example::
 For calling methods on objects, you can simplify yet further by using the
 :func:`getattr` built-in to retrieve methods with a particular name::
 
-   def visit_a(self, ...):
-       ...
-   ...
+   class MyVisitor:
```

Comment:
This case is special. There was no `class` context defined. I've added it as well.